src/app/clark.component.ts

```diff
@@ -92,10 +93,6 @@ export class ClarkComponent implements OnInit {
         }
       });
 
-    if (localStorage.getItem('cookieAgreement')) {
```

Comment:
Why was this removed?

Reply:
Everything relating to cookies is now handled in CookieAgreementService so this didn't serve a purpose anymore.